scripts/postgresql.conf

```diff
@@ -411,7 +411,7 @@ max_wal_senders = 0		# max number of walsender processes
 #geqo_threshold = 12
 #geqo_effort = 5			# range 1-10
 #geqo_pool_size = 0			# selects default based on effort
-#geqo_generations = 0			# selects default based on effort
+#geqo_genericerations = 0			# selects default based on effort
```

Comment:
Looks like this got blasted by a global search/replace

Reply:
Good catch, fixed